ui/components/ui/textarea/textarea.test.js

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import {
+  COLORS,
+  RESIZE,
+  SIZES,
+  BORDER_STYLE,
+} from '../../../helpers/constants/design-system';
+import TextArea from '.';
+
+describe('TextArea', () => {
+  const text =
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod temporld, Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod temporld, Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod temporld';
+  const onChange = jest.fn();
+  let args;
+  beforeEach(() => {
+    args = {
+      name: 'Text area',
+      value: text,
+      resize: RESIZE.BOTH,
+      scrollable: false,
+      boxProps: {
+        borderColor: COLORS.UI3,
+        borderRadius: SIZES.SM,
+        borderStyle: BORDER_STYLE.SOLID,
+        padding: [2, 4],
+      },
+      height: '100px',
+      onChange,
+    };
+  });
+  it('should render the TextArea component without crashing', () => {
+    const { getByText } = render(<TextArea {...args} />);
+    expect(getByText(text)).toBeDefined();
+  });
+  it('should call onChange when there is a change made', () => {
+    const { container } = render(<TextArea {...args} />);
+    fireEvent.change(container.firstChild, { target: { value: 'abc' } });
+    expect(onChange).toHaveBeenCalled();
```

Comment:
[nit] I think we ought to test that the input text is rendered rather than whether the onChange method was called

Reply:
Good suggestion, will make a note to make a PR for this in the future!